### ISSUE TITLE
Environment variable ... converted to str implicitly

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#4136]
```
.tox/py36/lib/python3.6/site-packages/_pytest/monkeypatch.py:234
  /home/travis/build/borgbackup/borg/.tox/py36/lib/python3.6/site-packages/_pytest/monkeypatch.py:234:
    PytestWarning: Environment variable value local('/tmp/pytest-of-travis/pytest-0/popen-gw2/xdg-config-home0') should be str, converted to str implicitly
    value
```
a flood of these is seen on travis-ci sometimes.

**Comment by step21** [comment]
The first one I found, but any specific command to see where exactly these variables originate? Somehow I find them hard to find in the code.

**Comment by ThomasWaldmann** [comment]
I currently only have what I posted above. Seen a flood of these in the test runs. From the sheer amount, I suspect it might be the test setup code (iirc in conftest.py).

**Comment by ThomasWaldmann** [comment]
maybe it helps to temporarily modify `monkeypatch` to just raise an Exception there instead of a warning, so one gets more information about where it goes wrong.

**Comment by ThomasWaldmann** [comment]
fixed by #4142.

btw, ending your PR or commit comment with ", fixes #nnnn" would close the issue nnnn automatically.

**Comment by step21** [comment]
Yeah, sorry, I forgot to add the #id here.
